gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -21,6 +21,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :Vm
 
   OrchestrationTemplateCfn.register_eligible_manager(self)
+  ExtManagementSystem.register_cloud_discovery_type('amazon' => 'ec2')
```

Comment:
the values are used here https://github.com/ManageIQ/manageiq/blob/355edd71280916cddfb1f0246ccbef853431157c/app/controllers/application_controller/ci_processing.rb#L768

until we come up with code to ask the provider wether a) it is discoverable and b) what it needs for it, I'd leave it like this. 
This way amazon would not show up in the discover dropdown